packages/jbrowse-cli/src/testUtil.ts

```diff
@@ -74,16 +69,17 @@ export const test = oclifTest
 export const setup = test
   .mockStdoutWrite()
   .add('originalDir', () => process.cwd())
-  .add('dir', async () => {
+  .add('dir', () => {
+    const tmpDir = fs.realpathSync(os.tmpdir())
```

Comment:
I don't understand what's going on here. How does running `os.tmpdir()` in each test change anything, when it always returns the same location?

Reply:
The main issue was unrelated to these so I'll revert